### PR TITLE
Fix news item count

### PR DESCRIPTION
**Pull request by ZachCMP**
- Actually caps the new items on the News and Publications page to 20
- Fixes the links on the homepage

### CODE REVIEW

File: app/views/root/news.html.slim [hunk 1]
```diff
@@ -14,7 +14,7 @@
       section.mt-6
         .container
           .row.o-card-row
-            - NewsItem.current.each_with_index do |item, i|
+            - NewsItem.current.limit(20).each_with_index do |item, i|
```
Comment (lbraun) [inline app/views/root/news.html.slim:17]: Does this have a sort on it to ensure we always get the same 20? I'm guessing yes.

File: test/system/root_test.rb [hunk 2]
```diff
@@ -27,6 +27,18 @@ def test_news_page
     assert page.current_path == '/archive'
   end
 
+  def test_news_page_limit
+    30.times { |n| create(:news_item, title: "News Item #{n}", published: true) }
```
Comment (lbraun) [inline test/system/root_test.rb:31]: You can also do `create_list` with FactoryBot, just FYI

**Review by lbraun — APPROVED** [review]
💯

File: app/views/root/news.html.slim (hunk 1 above)
Comment (ZachCMP) [reply app/views/root/news.html.slim:17]: Yes, `NewsItem.current` is sorted

**Pull request merged**